changelogs/fragments/64076-urls-timeout-parameter.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - core - remove uneeded Python version checks.
```

Sorry, missed a tiny detail :)
```suggestion
  - core - remove unneeded Python version checks.
```